flink-formats/flink-sql-protobuf/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,9 @@
+flink-sql-protobuf
+Copyright 2014-2021 The Apache Software Foundation
```

Comment:
```suggestion
Copyright 2014-2022 The Apache Software Foundation
```

I found that many modules did not update this to '2022', I've created FLINK-28676 to handle existing ones.